### ISSUE TITLE
Small accessibility enhancement on our Storefront

### ISSUE THREAD

**Issue by Febakke** [digdir/designsystemet#3317]
A collection of small fixes. 
- [x] Image has `alt: image` [on this page](https://www.designsystemet.no/grunnleggende/designelementer/ikoner) Can add alt text: "Skjermbilde som viser hvor i Figmas grensesitt du kan kjøre Navs plugin for å oppdatere ikonene" 
- [x] Language on code examples 🤔 Not sure what we should do with this. They are often a combination of english and norwegian. Investigate this one more.
- [x] Same link text different URL [Blog article](https://www.designsystemet.no/bloggen/2024/v1rc1) , [pattern article on form validation)](https://www.designsystemet.no/monstre/feilmeldinger) and [pattern article on system alers](https://www.designsystemet.no/monstre/systemvarsler) All of them are Github links. They should have different link texts.
- [x] Same link texts on our page about [design principles](https://www.designsystemet.no/grunnleggende/introduksjon/designprinsipper).  Designprinsiper and design principles are recuring multiple times
- [x] Links with `target='_blank'`  (Open in new tab) go trough and remove or add text in link where appropriate. Example from [Dette kommer i V1](https://www.designsystemet.no/bloggen/2024/v1rc1) This link and text is a little outdated as well.

![Image](https://github.com/user-attachments/assets/542f50b5-4c35-46f3-9afd-9508fa7ba5cd)

- [x] På designprinsipper er lenken til EnTur brutt. Bytt til: https://design.entur.no/kom-i-gang/introduksjon/designprinsipper
- [x] link to miro board on [insight hub article](https://www.designsystemet.no/god-praksis/brukerinnsikt/felles-innsiktsbase) is not functional. (I have reached out to see if this should be fixed or removed)

**Comment by Barsnes** [comment]
> Language on code examples 🤔 Not sure what we should do with this. They are often a combination of english and norwegian. Investigate this one more.

We will keep our `no` language, since any programming language is the same in different languages. Text we write ourselves should be norwegian.
